entity-services/src/main/xdmp/entity-services/entity-services.xqy

```diff
@@ -149,29 +152,29 @@ declare function es:extraction-template-generate(
  : Generates an element for configuring Search API applications
  : Intended as a starting point for developing a search grammar tailored
  : for entity and relationship searches.
- : @param An entity type document.
+ : @param An entity model
  :)
 declare function es:search-options-generate(
-    $model
+    $model as map:map
```

Comment:
Good catch.
Having map:map in here as an argument ensures that, if passed a JSON document or object node, the model will be automatically converted to a map:map.  

So this is a behavior change -- I had added ensure-model() to force users to pass documents to these functions, but that decision made the API hard to use for JavaScript folks.

So you will have some changes in those tests that verified

https://bugtrack.marklogic.com/38517

because now, it's perfectly OK to pass document nodes and elements and object nodes to model-validate